commands/exportcommands.py

```
@@ -767,33 +730,33 @@ def _create_pull_request(self):
-        print(f"Directory Local Export: {self.export_dir}")
+        print(f"📂 Local Export: {self.export_dir}")
         archive_files = [f for f in os.listdir(self.export_dir) if f.endswith('.tar.gz')]
         if archive_files:
-            print(f"Archive Archive: {archive_files[0]}")
-        print(f"Branch Branch: {self.export_branch}")
-        print(f"PR Pull Request: {pr_url}")
-        print(f"\nSummary Export Summary:")
+            print(f"📦 Archive: {archive_files[0]}")
+        print(f"🌟 Branch: {self.export_branch}")
+        print(f"🔗 Pull Request: {pr_url}")
+        print(f"\n📊 Export Summary:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)